components/Des.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+export default function Des(props) {
+    return (
+      <div>
+        <div className="game_desc">
+          <h2 className="game_title">{props.game}</h2>
+          <p className="desc">
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit,
+            sed do eiusmod tempor incididunt
+            ut labore et dolore magna aliqua.
+            Ut enim ad minim veniam, quis nostrud exercitation ullamco
+            laboris nisi ut aliquip ex ea commodo consequat.
+          </p>
+        </div>
+        <style jsx>
+        {`
+        
+        .game_title {
+          font-family: 'Orbitron';
+          font-size: 40px;
```

Comment:
normally font-size is styled by relative units like `rem` or `%`. It's better for responsive, but we can do this in another pull request.

A useful link https://www.w3schools.com/cssref/css_units.asp